src/CLI/commands/init.ts

```diff
@@ -178,6 +175,13 @@ async function init(argv: yargs.Arguments<InitOptions>, initMode: InitMode) {
 		extensions: path.join(cwd, ".vscode", "extensions.json"),
 	};
 
+	const templateDir = path.join(TEMPLATES_DIR, template);
+
+	const pathValues = Object.values(paths);
```

Comment:
This is created but never used. Did you forget to replace it in the iteration below?